results/client/client.go

```diff
@@ -0,0 +1,81 @@
+package client
+
+import (
+	"context"
+	"github.com/google/uuid"
+	proto "github.com/macformula/hil/results/client/generated"
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+type ResultsClient struct {
```

Comment:
can we add some brief comments to our structs and public methods?

Reply:
https://github.com/macformula/hil/issues/42